src/coreclr/src/jit/morph.cpp

```diff
@@ -16831,8 +16833,15 @@ void Compiler::fgMorphStructField(GenTree* tree, GenTree* parent)
                 tree->SetOper(GT_LCL_VAR);
                 tree->AsLclVarCommon()->SetLclNum(fieldLclIndex);
                 tree->gtType = fieldType;
+                bool canCSE  = tree->CanCSE();
+                // That doesn't make any sense, was it supposed to clean all flags except GTF_NODE_MASK here? 2012<
                 tree->gtFlags &= GTF_NODE_MASK;
+                // If yes then why a year after (2013) was this cleaning added, that obviously does nothing.
```

Comment:
I don't think that you want to checkin these two comments.

Reply:
Yes, I will drop that commit before merge, want to see if somebody knows more about these changes.